pcbuilder_crawlers/template.py

```diff
@@ -13,6 +15,17 @@ def translate_category(self, name):
             categoryArray = [i.encode('UTF-8') if isinstance(i, basestring)
                              else i for i in categories[category_names]]
             if name in categoryArray:
-                print 'found'
                 return categoryArray[0]
         return None
+
+    def translate_item(self, template, item):
+        for row in item:
+            for other_row in template:
+                encoded_template_row = [i.encode('UTF-8')
```

Comment:
commenting, I have no idea what's going on here or why it's happening :confused: